package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Browserify middleware for adding required styles to the page.",
   "main": "index.js",
   "scripts": {
-    "test": "browserify -t . -r ./index.js:sassify test/*.js -o test_pack.js && cat test_pack.js | tape-run -b firefox | tap-spec && cat test_pack.js | tape-run -b phantom | tap-spec"
+    "test": "browserify -t . -r ./index.js:sassify test/*.js -o test_pack.js && cat test_pack.js | tape-run -b firefox | tap-spec && cat test_pack.js | tape-run -b phantom | tap-spec",
+    "test-importer": "node_modules/.bin/mocha test/importer.js"
```

Comment:
Didn't want to make too many assumptions here. Is mocha cool? would you prefer this command be added to the test script?